gpustack/policies/candidate_selectors/vllm_resource_fit_selector.py

```diff
@@ -70,11 +70,30 @@ async def estimate_model_vram(model: Model) -> int:
     return weight_size * 1.2 + cuda_graph_size
 
 
+def get_model_num_attention_heads(model: Model) -> int:
+    """
+    Get the number of attention heads in the model.
+    """
+
+    num_attention_heads = None
+    try:
+        config = get_pretrained_config(model, trust_remote_code=True)
```

Comment:
Why true v.s. getting from the parameters?

Reply:
without setting parameter will fail to get the config